src/test/resources/features/04_employment.feature

```diff
@@ -1729,3 +1729,294 @@ Feature: Employment Journeys - Income Tax Submission
+    And the user clicks on the change link
+    Then the user should be redirected to the "Employment details and benefits" page
+    And the user clicks on the expenses link
+
+    Then the user should be redirected to the "Check your client’s employment expenses" page
+    When the user clicks on the first change employment expenses link
+    Then the user should be redirected to the "Do you want to claim employment expenses for your client?" page
+    When the user selects the yes radio button
+    When the user clicks the continue button
+    Then the user should be redirected to the "Employment expenses" page
+    When the user clicks the continue button
+
+    Then the user should be redirected to the "Do you want to claim your client’s business travel and overnight expenses?" page
+    When the user selects the yes radio button
+    When the user clicks the continue button
+    Then the user should be redirected to the "How much do you want to claim for your client’s business travel and overnight stays?" page
+    When the user selects the amount field and enters a value of 100.11
+    And the user clicks the continue button
+
+    Then the user should be redirected to the "Do you want to claim for uniforms, work clothes, or tools for your client?" page
+    When the user selects the yes radio button
+    When the user clicks the continue button
+    Then the user should be redirected to the "How much do you want to claim for uniforms, work clothes, or tools for your client?" page
+    When the user selects the amount field and enters a value of 200.22
+    And the user clicks the continue button
+
+    Then the user should be redirected to the "Do you want to claim for professional fees and subscriptions for your client?" page
+    When the user selects the yes radio button
+    When the user clicks the continue button
+    Then the user should be redirected to the "How much do you want to claim for professional fees and subscriptions for your client?" page
+    When the user selects the amount field and enters a value of 300.33
+    And the user clicks the continue button
+
+    Then the user should be redirected to the "Do you want to claim for buying other equipment for your client?" page
+    When the user selects the yes radio button
+    When the user clicks the continue button
+    Then the user should be redirected to the "How much do you want to claim for buying other equipment for your client?" page
+    When the user selects the amount field and enters a value of 400.44
+    And the user clicks the continue button
+
+    Then the user should be redirected to the "Check your client’s employment expenses" page
+    And the user clicks the save and continue button
+    Then the user should be redirected to the "Employment" page
+
+  Scenario: Individual user as a new submission, answers no to the Do you want to claim employment expenses?, straight to expenses check your answers.
```

Comment:
Think this is the same scenario as "Individual user as a new submission selecting no to do you want to claim expenses and to skip all expenses pages" <- this one has a better/ more succinct Scenario name

Reply:
It is a bit different
One navigates from the CYA page to that link (as prior data) and one navigates from a new submission flow (interrupts page) which is why I included both